modules/wsl-distro.nix

```
@@ -129,6 +129,13 @@ with builtins; with lib;
           hosts.enable = false;
           "resolv.conf".enable = false;
         };
+
+        systemPackages = [
+          (pkgs.runCommand "wslpath" { } ''
+            mkdir -p $out/bin
+            ln -rs /init $out/bin/wslpath
```

```suggestion
            ln -s /init $out/bin/wslpath
```

We don't want a relative symlink here.